server/data/nomisClientBuilder.js

```diff
@@ -58,6 +58,22 @@ module.exports = function(token) {
         getHdcEligiblePrisoners: function() {
             const path = `${apiUrl}/users/me/hdc-eligible`;
             return nomisGet(path, '', token);
+        },
+
+        getImageData: async function(id) {
+            try {
+                const result = await superagent
+                    .get(`${apiUrl}/images/${id}/data`)
+                    .set('Authorization', `Bearer ${generateApiGatewayToken()}`)
+                    .set('Elite-Authorization', token)
+                    .responseType('blob');
+
```

Comment:
could call nomisGet with an extra param for responseType

Reply:
Agreed.